src/WaterPokemon.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class WaterPokemon extends Pokemon{
+
+    String[] attacks = {"surf" ,"hydroPump" ,"hydroCanon" ,"rainDance"};
+
+    public WaterPokemon(String name, int level, int hp, String food, String sound) {
+        super("water",name, level, hp, food, sound);
+    }
+
+    public List<String> getAttacks(){
+        ArrayList<String> strings = new ArrayList<>(Arrays.asList(attacks));
+        return strings;
+    }
+
+    public void surf(Pokemon pokemon, Pokemon enemy){
```

Comment:
Het is jammer dat je de aanvallen nog niet geimplementeerd hebt. Misschien dat je hier later nog tijd voor kunt vinden